code/datums/mood_events/generic_positive_events.dm

```diff
@@ -24,6 +24,11 @@
 	mood_change = 1
 	timeout = 2 MINUTES
 
+/datum/mood_event/cuteheadpat
+	description = "<span class='nicegreen'>That was such a cute head pat!</span>\n"
```

Comment:
Kinda odd that you're describing a tactile sensation with visual descriptors, especially since the person receiving the head pat isn't really seeing it? Something like "Wow, that head pat was nice!" or something else to indicate that the high-tech gloves (that fry brains when emagged) are doing some sort of tactile tech-magic.

Reply:
yeah i couldn't think of a good descriptor at the time or writing that, thanks for the advice